src/classes/Table/Table.ts

```diff
@@ -1187,16 +1188,16 @@ class Table<Name extends string, PartitionKey extends A.Key, SortKey extends A.K
     } // end-if
   } // end put
 
-  private parseBatchWriteResponse(result: any, options: batchWriteOptions = {}): any {
+  private parseBatchWriteResponse(result: PromiseResult<BatchWriteItemOutput, AWSError>, options: batchWriteOptions = {}) {
     return Object.assign(
       result,
       // If UnprocessedItems, return a next function
       result.UnprocessedItems && Object.keys(result.UnprocessedItems).length > 0
         ? {
-            next: async () => {
+            next: async (): Promise<any> => {
               const nextResult = await this.DocumentClient!.batchWrite(
                 Object.assign(
-                  { RequestItems: result.UnprocessedItems },
+                  { RequestItems: result.UnprocessedItems! },
```

Comment:
Valid because of the ternary on [L1195](https://github.com/jeremydaly/dynamodb-toolbox/pull/337/files#diff-b79cb0aea0d1fa41dc3edc2eb9f58306c6c7b0a2304162cf5a493414d0492efbR1195).